src/main/java/edu/ie3/datamodel/utils/TimeSeriesUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.utils;
+
+import edu.ie3.datamodel.models.timeseries.individual.IndividualTimeSeries;
+import edu.ie3.datamodel.models.value.Value;
+import edu.ie3.util.interval.ClosedInterval;
+import java.time.ZonedDateTime;
+import java.util.stream.Collectors;
+
+public class TimeSeriesUtil {
```

Comment:
I would expect this is a util class? Can be static? Private constructor is missing here as well

Reply:
![](https://media.giphy.com/media/ylPWDQuapyexa/source.gif)

Reply:
There is no such thing like a static outer class, as it effectively is static right from the start. :smiley: